b-lab-org/ember-cli-impact-core
# [Wellness] Upgrade Ember dependencies to 3.x [#155750558]

## Overview
Moving most major Ember dependencies to `3.x` as well as some additional wellness.

## What Changed
- Ember dependencies updated to latest
- Bower has been removed entirely
- `chosen` has been removed entirely
- `eslint` overrides cleaned up

## How to test
The biggest change here is that we no longer have a `chosen` UI for `impact-question-select` type questions. You'll encounter this most commonly on the questions "Select your operating currency", which will now just be a dropdown instead of a chosen dropdown.

There are some other changes for consuming applications, which will be QA'd independently.

Some regression around the dummy app will be sufficient. More QA will be done in the `impact-app` and `manage-app`.
